projects/ui/src/Utility/utility.ts

```diff
@@ -98,3 +101,13 @@ export function getEnumValues<Enum>(pEnum: StandardEnum<Enum>): Enum[] {
 export function capitalize(input: string) {
   return input[0].toUpperCase() + input.substring(1);
 }
+
+export function omitErrorMessageResponse<T>(value: T | ErrorMessageResponse) {
+  if (value === null) {
+    return null;
+  }
+  if (typeof value === "object" && "message" in value) {
```

Comment:
The backend error messages come in like that (with `"message"`), so the `"message"` is here since it's sometimes just passing that value through. But I'm sure there is a better way to check for it.
 
Something that was kind of inconsistent that this tries to address is that `toast.Promise(...)`, expects an error to be thrown, whereas other places don't expect that. Ideally the errors would all be dealt with the same way, so I'll look into that (and this) some more when I'm refactoring later.